pages/take-action.vue

```diff
@@ -124,6 +133,32 @@ export default Vue.extend({
 .v-application--wrap {
   justify-content: space-between;
 }
+
+.Take-Action-Title{
+  font-family: Avenir;
+  font-style: normal;
```

Comment:
nit: I think we already have the following as default styles:
```
font-family: Avenir;
font-style: normal;
```

So the only styling you need should be `font-weight`
```suggestion
```